scripts/uninstall_debs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+dpkg -r openrazer-meta
+dpkg -r python3-openrazer
+dpkg -r openrazer-daemon
+dpkg -r openrazer-doc
+dpkg -r openrazer-driver-dkms
```

Comment:
Perhaps add a `sudo` prefix? It seems unlikely that users will be logged in as root (or might forget `sudo ./uninstall_debs.sh`)

```suggestion
sudo dpkg -r openrazer-meta
sudo dpkg -r python3-openrazer
sudo dpkg -r openrazer-daemon
sudo dpkg -r openrazer-doc
sudo dpkg -r openrazer-driver-dkms
```

Reply:
fixed